ethereum/evmtool/src/test/resources/org/hyperledger/besu/evmtool/t8n/prague-withdrawal-request.json

```diff
@@ -131,7 +133,7 @@
   "stdout": {
     "alloc": {
       "0x0000000000000000000000000000000000001000": {
-        "code": "0x60386000600037600060006038600060017309fc772d0857550724b07b850a4323f39112aaaa620f4240f150",
+        "code": "0x6038600060003760006000603860006001730c15f14308530b7cdb8460094bbb9cc28b9aaaaa620f4240f150",
```

Comment:
proxy contract calls the withdrawal request address so it's updated here in the code also